pkg/realmd/operation.html

```diff
@@ -13,10 +13,14 @@ <h4 class="modal-title realms-op-title"></h4>
                 <input class="realms-op-field form-control realms-op-address" type="text" value=""/>
               </td>
               <td style="width:0px">
-                <div class="realms-op-address-spinner spinner"
-                  style="margin-left:-40px;margin-top:4px" hidden></div>
-                <img class="realms-op-address-error" src="images/dialog-error.png"
-                  style="margin-left:-40px;margin-top:4px" hidden/>
+                <div class="realms-op-address-spinner"
+                     style="margin-left:-30px;margin-top:2px" hidden>
```

Comment:
We should be using CSS classes for this. In fact there's already a class. And since we're changing them, I think we should put them in classes at this point.